mmaction/datasets/pipelines/augmentations.py

```diff
@@ -134,38 +134,58 @@ def __init__(self, transforms):
                 [self.imgaug_builder(t) for t in self.transforms])
 
     def default_transforms(self):
-        """Default transforms for imgaug."""
+        """Default transforms for imgaug.
+
+        Implement RandAugment by imgaug.
+        Plase visit `https://arxiv.org/abs/1909.13719` for more information.
+
+        Augmenters and hyper parameters are borrowed from the following repo:
+        https://github.com/tensorflow/tpu/blob/master/models/official/efficientnet/autoaugment.py # noqa
+
+        Miss one augmenter ``SolarizeAdd`` since imgaug doesn't support this.
+
+        Returns:
+            dict: The constructed RandAugment transforms.
+        """
+        # RandAugment hyper params
+        n, m, max_level = 2, 9, 10
+        cur_level = 1.0 * m / max_level
 
         return [
-            dict(type='Rotate', rotate=(-30, 30)),
             dict(
                 type='SomeOf',
-                n=(0, 3),
+                n=n,
                 children=[
                     dict(
-                        type='OneOf',
-                        children=[
-                            dict(type='GaussianBlur', sigma=(0, 0.5)),
-                            dict(type='AverageBlur', k=(2, 7)),
-                            dict(type='MedianBlur', k=(3, 11))
-                        ]),
+                        type='ShearX',
+                        shear=17.19 * cur_level * random.choice([-1, 1])),
+                    dict(
+                        type='ShearY',
+                        shear=17.19 * cur_level * random.choice([-1, 1])),
+                    dict(
+                        type='TranslateX',
+                        px=int(100 * cur_level * random.choice([-1, 1]))),
```

Comment:
What's the assumption on input image size?

Reply:
In official repo, when training ImageNet with RandAugment, the image sizes for ResNet/EfficientNet-B5/EfficientNet-B7 are 224/456/600.

Actually in other implementations of RandAugment, TranslateX/TranslateY use percentage params, such as 20%.

Thanks for pointing out.

Reply:
Video frames are generally much smaller than image

Reply:
Done, use `percent` instead of `px`